project/Versions.scala

```diff
@@ -2,7 +2,7 @@
 object Versions {
   val AsyncHttpClientVersion = "2.12.3"
   val AWSVersion             = "1.11.751"
-  val AWSSdk2Version         = "2.17.100"
+  val AWSSdk2Version         = "2.16.13"
```

Comment:
For the context: there were issues with accessing S3 from the Fargate ECS instance. The easiest fix for now is to rollback dependencies to the last and the most tested version (the GeoTrellis AWS SDKv2 version).

Update of this library requires a little bit of more testing, mb they just slightly changed the API or it is in conflict with the GeoTrellis transitive dep. A link to [GT Versions.scala](https://github.com/locationtech/geotrellis/blob/v3.6.0/project/Dependencies.scala#L86)

@aaronxsu could you add smth like that into the project/Versions.scala? + a note into the CHANGELOG.